### DISCUSSION TITLE
Ask for confirmation less often

### DISCUSSION THREAD

**Discussion by colinfrei** [jonaswinkler/paperless-ng#1174] [Feature Requests]
It'd be nice of Paperless wouldn't ask for confirmation as often as it does when changing tags, correspondent, ...
What would be nice is if it'd show a message that it changed things, with an undo link instead.
Alternatively, or in addition, it might be possible to only show the confirmation prompt if I'm making changes that would hide documents from the current view. So if I'm looking at my inbox documents, removing the inbox tag would ask for confirmation, but setting a correspondent would not. The reasoning behind that is that it's easier to undo something manually if you still see the documents in question.

**Comment by beardedfool** [comment]
Just wanted to check you've seen settings> show confirmation dialogues?

**Reply by colinfrei** [reply]
Actually, I did not - thanks!
That's a bit less elegant, but more than good enough for me, so this can be closed.

Thanks again!
